modules/Hoot/ui/conflicts/merge.js

```diff
@@ -214,31 +213,31 @@ export default class Merge {
     }
 
     /**
-     * Generate metadata for merged node
+     * Generate metadata for merged feature
      *
-     * @param mergedNodeId - node ID
+     * @param merged - merged feature
      * @param relationId - relation ID
-     * @param mergedIdx - index of node in relation
+     * @param mergedIdx - index of merged feature in relation
      */
-    createNewRelationNodeMeta( mergedNodeId, relationId, mergedIdx ) {
-        let node = new osmNode(),
+    createNewRelationMember( merged, relationId, mergedIdx ) {
+        let member = {},
```

Comment:
@jackgrossman18 we don't need to treat this var as an osm node or way entity as it's never used as such, so this will allow the code to be generic for poi->poi and poi->poly

The rest of the changes just replace the assumption of dealing with a node to a generic feature